src/getCookieCutterSource.js

```diff
@@ -49,6 +51,20 @@ const getCookieCutterSource = async ({ branchName, destination }) => {
     source: destinationSourceFilename,
     destination,
   });
+
+  /*
+    Code is cloned inside "frontend-cookiecutter-application-master".
+    This extracts the content files to an upper level so the code
+    will be created in the folder you indicated
+  */
+  await move(`${destination}/frontend-cookiecutter-application-master/*`, destination, { dot: true })
+    .then(() => 'Content of node_modules moved successfully!')
+    .catch((error) => {
+      console.error(error);
+    });
```

Comment:
Let's avoid adding new dependencies if we can help it. fs-extra is already included and I think can get the job done. Something like this:

```
  fse.moveSync(`${destination}/frontend-cookiecutter-application-master`, destination, { overwrite: true });
```

Reference: https://github.com/jprichardson/node-fs-extra/blob/master/docs/move-sync.md

Reply:
Thanks for the feedback.. I needed to create a helper function because I needed to move the directory content, not the directory itself